server/src/main/java/org/apache/iotdb/db/qp/physical/sys/SetStorageGroupPlan.java

```diff
@@ -57,6 +58,15 @@ public void setPath(Path path) {
     return ret;
   }
 
+  @Override
+  public List<String> getPathsStrings() {
+    List<String> ret = new ArrayList<>();
+    if (path != null) {
+      ret.add(path.getFullPath());
+    }
+    return ret;
```

Comment:
And here.

Reply:
I'll change it. Thanks for your suggestion.